payments-core/src/test/java/com/stripe/android/polling/DefaultIntentStatusPollerTest.kt

```diff
@@ -40,6 +40,8 @@ class DefaultIntentStatusPollerTest {
 
         advanceTimeBy(nextDelay())
         assertThat(poller.state.value).isEqualTo(Succeeded)
+
+        poller.stopPolling()
```

Comment:
How come these needed to change?

Reply:
That’s because we’re no longer wrapping `StripeRepository.retrievePaymentIntent()` in a `runCatching` block.

Previously, we could call `removeFirst()` on an empty `queue` because the `runCatching` wrapper around `retrievePaymentIntent()` would make sure that we didn’t throw an exception. We basically swallowed that exception.

However, as a result of my changes, the `removeFirst()` call inside `FakeStripeRepository` is no longer wrapped in the same way, meaning that illegitimate calls are actually throwing exceptions.